packages/flutter/test/painting/image_resolution_test.dart

```diff
@@ -44,7 +44,7 @@ void main() {
 
       assetBundleMap[mainAssetPath] = <String>[];
 
-      final AssetImage assetImage = AssetImage(
+      final io.AssetImage assetImage = io.AssetImage(
```

Comment:
Is this because the web asset image doesn't work with the `TestAssetBundle`?

Reply:
This is interestingly due to a lint regarding calling a method called protected outside of a subclass